core/src/test/scala/unit/kafka/log/LogTest.scala

```diff
@@ -37,14 +38,14 @@ class LogTest extends JUnitSuite {
 
   @Before
   def setUp() {
-    logDir = TestUtils.tempDir()
+    logDir = TestUtils.tempPartitionLogDir(tmpDir)
```

Comment:
It may be worth initialiasing logDir in the class body for consistency with tmpDir initialisation. There are a few other examples like this.